packages/@aws-cdk/aws-codedeploy/lib/deployment-group.ts

```diff
@@ -98,6 +98,39 @@ class ImportedServerDeploymentGroupRef extends ServerDeploymentGroupRef {
   }
 }
 
+/**
+ * Represents a group of instance tags.
+ * An instance will match a group if it has a tag matching
+ * any of the group's tags by name and any of the provided values -
+ * in other words, tag groups follow 'or' semantics.
+ * If the value for a given key is an empty array,
+ * an instance will match when it has a tag with the given key,
+ * regardless of the value.
+ * If the key is an empty string, any tag,
+ * regardless of its key, with any of the given values, will match.
+ */
+export interface InstanceTagGroup {
```

Comment:
Don't know if this is going to translate well over jsii. I think it won't. Better definition would be:

```ts
export type InstanceTagGroup = {[key: string]: string[]};
```

Reply:
Sure. Will change.